tests/server/api/v1/test_records.py

```diff
@@ -145,3 +158,72 @@ def test_create_record_response_with_nonexistent_record_id(client: TestClient, d
 
     assert response.status_code == 404
     assert db.query(Response).count() == 0
+
+
+def test_list_record_responses(client: TestClient, db: Session, admin: User, admin_auth_header: dict):
```

Comment:
I believe you are not using `admin` in this test:

```suggestion
def test_list_record_responses(client: TestClient, db: Session, admin_auth_header: dict):
```